steps/run/Dockerfile

```diff
@@ -1,8 +1,7 @@
-FROM relaysh/core:latest
-RUN apk --no-cache add bash ca-certificates jq
-COPY step.sh /relay/step.sh
-CMD ["/relay/step.sh"]
+FROM relaysh/core:latest-python
+COPY step.py /relay/step.py
+CMD /usr/local/bin/python /relay/step.py
 
 LABEL "org.opencontainers.image.title"="workflow-run"
-LABEL "org.opencontainers.image.description"="Run another Relay workfow"
+LABEL "org.opencontainers.image.description"="Run another Relay workfow (python version)"
```

Comment:
Tiny nit, but I don't think you need to call out the implementation of the step really. We don't currently use this label anywhere but I think it might be confusing to see when it doesn't change the functionality.